plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointEntryIterator.java

```diff
@@ -707,6 +720,11 @@ private interface CheckPointFieldExtractor
          * checkpoint filter criteria.
          */
         @Nullable
-        DeltaLakeTransactionLogEntry getEntry(ConnectorSession session, Block block, int pagePosition);
+        DeltaLakeTransactionLogEntry getEntry(ConnectorSession session, int pagePosition, Block... blocks);
+
+        default int getRequiredChannels()
```

Comment:
I don't see usage of this method in code up to this commit, add it in the commit where it is used